### PR TITLE
My-Assignment-1-submission

### PR DESCRIPTION
**Pull request by rskadam2354**

### CODE REVIEW

**Review by sivasankar-v — APPROVED** [review]
Looking Nice

**Comment by rskadam2354** [comment]
> Looking Nice

should I merge it then ?

File: src/Assignment-1/assignment1.html [hunk 1]
```diff
@@ -14,33 +14,8 @@
...
-				<li>List 2</li>
-				<li>List 3</li>
-				<ul>
-					<li>List 3.1</li>
-					<li>List 3.2</li>
-				</ul>
-
-			</ul>
-		</div>
-		<div class="column">
```
Comment (ritu121) [inline src/Assignment-1/assignment1.html:32]: For Column keep class name as a col

File: src/Assignment-1/assignment1.html [hunk 2]
```diff
@@ -14,33 +14,8 @@
 	<!-- This is just a comment. Ignore this line.-->
 	<!-- Start your code from here -->
 	
-	<!-- this is changing -->
+	
 	<!-- End your code -->
-	<div class="row">
-		<div class="column">
```
Comment (ritu121) [inline src/Assignment-1/assignment1.html:20]: Please keep class name for all column as col..so that it will help for responsiveness of your webpage

**Comment by ritu121** [comment]
Looks good

**Pull request merged**